src/crimson/os/seastore/device.h

```diff
@@ -25,7 +25,7 @@ struct device_spec_t {
     denc(v.id, p);
     DENC_FINISH(p);
   }
-} __attribute__ ((packed));
+};
```

Comment:
> device_spec_t should not include denc and be packed at the same time

if we only need one of them, why they are both dropped in this change?

Reply:
I think we should either encode and decode (`denc()`) `device_spec_t`, or pack it. Only `__attribute__ ((packed))` is droped for `device_spec_t`.

Reply:
ah, sorry, i was wrong. should have read the commit message more carefully.